flowplayer.desktop

```diff
@@ -6,4 +6,8 @@ Exec=flowplayer
 Name=FlowPlayer
 
 [X-Sailjail]
-Sandboxing=Disabled
+OrganizationName=sailfishos-applications
+# ApplicationName does not have to be identical to Name
+ApplicationName=flowplayer
+# Add the required permissions here
+Permissions=UserDirs;Audio;MediaIndexing;Bluetooth;Internet;RemovableMedia
```

Comment:
Let us hope that this really does not break anything, though I cannot think of any property missing here.
<sub>I really would have appreciated this mini-change in a separate PR, which I would have merged later (after the next release).  Never mind, this sleight concern is not worth the effort to separate that now.</sub>

Reply:
Yes, I was puzzling about this. What it boils down to is that the changes should be Sailjail compatible.  So, I implemented them that way. That meant the old method had to go. Whereupon I tested with Sailjail.